### PR TITLE
New Adapter: SUNT Content

### PR DESCRIPTION
**Pull request by sag-celejd**
https://github.com/prebid/prebid.github.io/pull/4076

Note:
The implementation of the adapter is basically identical to the Seeding Alliance Adapter since we are the maintainers of both adapters.

### CODE REVIEW

File: adapters/suntContent/suntContent_test.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+	}, config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})
+	if buildErr != nil {
+		t.Fatalf("Builder returned unexpected error %v", buildErr)
+	}
+
+	adapterstest.RunJSONBidderTest(t, "suntContenttest", bidder)
+}
+
+func TestResolvePriceMacro(t *testing.T) {
+	_, buildErr := Builder(openrtb_ext.BidderSuntContent, config.Adapter{
```
Comment (hhhjort) [inline adapters/suntContent/suntContent_test.go:27]: Why are you invoking the builder in all the tests when you don't seem to be using it?

File: static/bidder-info/suntContent.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+endpoint: "https://b.suntcontent.se/cds/rtb/bid?ssp=pbs"
+maintainer:
+  email: tech@sunt
```
Comment (hhhjort) [inline static/bidder-info/suntContent.yaml:3]: This does not look valid. Is it missing a `.com` or something at the end?

File: adapters/suntContent/suntContent_test.go (hunk 1 above)
Comment (sag-celejd) [reply adapters/suntContent/suntContent_test.go]: Haha, no idea actually - probably just copied the example from the doc and underscored the unused variables at some point.

I removed the redundant parts in the Seeding Alliance and SUNT adapter.

**Comment by bsardo** [comment]
Hi @sag-celejd, I just noticed your tests are failing because your adapter still has a reference to the mxmcherry version of the ORTB library (`"github.com/mxmCherry/openrtb/v16/openrtb2"`).
We recently ported the mxmcherry version to prebid and merged that change into master so you'll need to reference the new library (`"github.com/prebid/openrtb/v17/openrtb2"`).

**Comment by hhhjort** [comment]
Is Sunt Content just a copy of Seeding Alliance? If so you don't need a second adapter in the adapters directory. You should be able to just use the seedingalliance builder a second time to build the Sunt Content adapter, giving it its own name and bidder-info yaml file. This will save having to maintain two copies of the adapter code.

**Comment by sag-celejd** [comment]
@hhhjort Thank you for your suggestion. We'd rather keep them separate since we will have independent changes in the future.

**Comment by hhhjort** [comment]
> @hhhjort Thank you for your suggestion. We'd rather keep them separate since we will have independent changes in the future.

Are the planned changes beyond the scope of what could be specified in the bidder yaml files? The yaml files will still be independent even if they share the same underlying bidder code. Otherwise separate adapters may be the best way to go.

**Comment by sag-celejd** [comment]
> Are the planned changes beyond the scope of what could be specified in the bidder yaml files?
Unfortunately yes.

File: adapters/suntContent/suntContent.go [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+	}
+
+	if bidExt.Prebid == nil {
+		return "", fmt.Errorf("bid.Ext.Prebid is empty")
+	}
+
+	return openrtb_ext.ParseBidType(string(bidExt.Prebid.Type))
+}
+
+func curExists(cc []string, c string) bool {
```
Comment (AlexBVolcy) [inline adapters/suntContent/suntContent.go:121]: Could you give more descriptive names for `cc` and `c`

File: adapters/suntContent/suntContent_test.go [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+			wantErrContain: "bid.Ext.Prebid is empty",
+			bidType:        "",
+		},
+	}
+
+	for _, test := range tests {
+		var bid openrtb2.SeatBid
+		var extBid openrtb_ext.ExtBid
+
+		var js string
```
Comment (AlexBVolcy) [inline adapters/suntContent/suntContent_test.go:109]: Could you make this variable name more descriptive

File: adapters/suntContent/suntContent.go (hunk 3 above)
Comment (sag-celejd) [reply adapters/suntContent/suntContent.go]: Sure, https://github.com/prebid/prebid-server/pull/2420/commits/0828e02dbc4b320282802b07ecebf1c00eea2c73

**Review by hhhjort — APPROVED** [review]

**Review by AlexBVolcy — APPROVED** [review]

**Pull request merged**